modules/microshift-install-multus-running-cluster.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-cni-multus.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-multus-installing-on-running-cluster_{context}"]
+= Installing the Multus CNI plugin on an running cluster
+
+If you want to attach additional networks to a pod for high-performance network configurations, you can install the {microshift-short} Multus RPM package. After installation, a host restart is required to recreate all the pods with the Multus annotation.
+
+[IMPORTANT]
+====
+The Multus RPM should not be removed after you install it.
+====
+
+.Procedure
+
+. Install the Multus RPM package by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install microshift-multus
+----
++
+[TIP]
+====
+If you create your custom resources (CRs) before completing your installation of Multus, you can avoid restarting the service a second time to apply them.
+====
+
+. To apply the manifest from the package to an active cluster, restart the host by running the following command:
++
+[source,terminal]
+----
+$ sudo systemctl restart
+----
+
+.Verification
+
+In general, any non-empty output is expected with this step.
```

Comment:
The verification steps and output is not clear for the user. Should we say both multus pod and multus dhcp-daemon will be created after systemctl restarting. And include below output
```
$ oc get pod -A | grep multus
openshift-multus                       dhcp-daemon-ktzqf                         1/1     Running   0          45h
openshift-multus                       multus-4frf4                              1/1     Running   0          45h

```
@pmtk WDYT?

Reply:
That looks better to me from a user perspective. We're just checking the packages at this point, not any config.

Reply:
@pmtk I'll put this PR in peer review; if you aren't happy with this update, LMK.